es-vue-base/build/rollup.config.js

```diff
@@ -101,17 +108,73 @@ const globals = {
     'bootstrap-vue': 'bootstrapVue',
 };
 
-// Customize configs for individual targets
+// UMD Build: https://github.com/umdjs/umd
 const buildFormats = [];
+if (!argv.format || argv.format === 'umd') {
+    const umdConfig = {
+        ...baseConfig,
+        external,
+        output: {
+            ...baseConfig.output,
+            format: 'umd',
+            globals,
+        },
+        plugins: [
+            replace(baseConfig.plugins.replace),
+            ...baseConfig.plugins.preVue,
+            vue(baseConfig.plugins.vue),
+            ...baseConfig.plugins.postVue,
+            babel({
```

Comment:
Naively I thought transpilation could happen in down-stream projects. Is the problem there that babel doesn't transpile `node_module` packages by default?

Reply:
That's right it ignores everything by default and you have to tell it what packages in `node_modules` you want to transpile.